homeassistant/components/homekit/accessories.py

```diff
@@ -459,8 +449,10 @@ async def async_call_service(self, domain, service, service_data, value=None):
         context = Context()
 
         self.hass.bus.async_fire(EVENT_HOMEKIT_CHANGED, event_data, context=context)
-        await self.hass.services.async_call(
-            domain, service, service_data, context=context
+        self.hass.async_create_task(
+            self.hass.services.async_call(
```

Comment:
You can await this. It defaults to blocking=False. 

https://github.com/home-assistant/core/blob/dev/homeassistant/core.py#L1467-L1469

Reply:
Oh I see now. Previously it would be passed to add_job. It's not called from an async function.

Reply:
Hmm.. Its already in a callback so I can't `await`

Reply:
It a callback from a `HAP-python` setter